<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptWatchControlUi</class>
 <widget class="QWidget" name="ScriptWatchControlUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Script Watches</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QToolButton" name="addWatchButton">
       <property name="toolTip">
        <string>Add Watch</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
      <item>
      <widget class="QToolButton" name="deleteWatchButton">
       <property name="toolTip">
        <string>Delete Watch</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="deleteAllWatchesButton">
       <property name="toolTip">
        <string>Delete All Watches</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="view">
     <property name="indentation">
      <number>10</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../ui/resources/wizard.qrc"/>
 </resources>
 <connections/>
</ui>
